app/controllers/clone_account_memberships_controller.rb

```diff
@@ -0,0 +1,74 @@
+class CloneAccountMembershipsController < ApplicationController
+
+  admin_tab :all
+  before_action :authenticate_user!
+  before_action :check_acting_as
+  before_action :init_current_facility
+  before_action :init_user
+  before_action :user_to_clone, only: :new
+  before_action :account_users, only: :new
+  before_action { @active_tab = "accounts" }
+
+  layout -> { request.xhr? ? false : "two_column" }
+
+  def index
+  end
+
+  def search
+    @users, _count = UserFinder.search(params[:search_term])
+
+    render :search
+  end
+
+  def new
+  end
+
+  def create
+    cloner = account_membership_cloner
+
+    if cloner.perform
+      flash[:notice] = t(:success, scope: "forms.clone_account_membership")
+
+      redirect_to facility_user_accounts_path
+    else
+      flash[:error] = t(
+        :error,
+        scope: "forms.clone_account_membership",
```

Comment:
We can probably consider using the `text` helper which gives us some some implicit scoping. Not a big deal, though.